pattern/index.js

```diff
@@ -46,14 +69,16 @@ function generatorPattern() {
     };
     var destinationPath = "src/" + generator.options.hierarchy + "/" + options.name;
 
-    generator.fs.copyTpl(generator.templatePath("**/*"), generator.destinationPath(destinationPath), options);
+    generator.fs.copyTpl(generator.templatePath("pattern.md"), generator.destinationPath(destinationPath + ".md"), options);
```

Comment:
Copied the files specifically.  Without this change the patterns were actually going into a folder with the right name, but the files were named `patterns.md` and `patterns.scss`